lib/glimesh_web/templates/layout/root.html.leex

```diff
@@ -62,7 +62,7 @@
         </div>
         <div class="col-md-9">
           <div class="row">
-            <div class="col-6 col-md-3 offset-md-3">
+            <div class="col-6 col-md-3 offset-md-2">
               <h5>Social</h5>
```

Comment:
To resolve the current conflict you'll just have to do:
`<h5><%= dgettext("footer", "Social") %> </h5>`